src/pages/TrainingPlan/components/TrainingBlock.jsx

```diff
@@ -0,0 +1,84 @@
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { Box, Card, CardContent, Collapse, Typography } from "@mui/material";
+
+import { useTheme } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+import CircularProgressWithLabel from "../../../components/CircularProgressWithLabel";
+import ExpandMoreButton from "../../../components/ExpandMoreButton";
+
+/**
+ * A component that displays a training block with course name, skills, progress, and description.
+ *
+ * @param {Object} props - The props object.
+ * @param {string} props.courseName - The name of the course.
+ * @param {string[]} props.skills - An array of skills related to the course.
+ * @param {string} props.description - The description of the course.
+ * @param {number} props.progress - The progress of the course as a percentage.
+ * @returns {JSX.Element} - The TrainingBlock component.
+ */
+
+function TrainingBlock(props) {
+  const theme = useTheme();
+
+  const lightBlue = theme.palette.primary.light;
+
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <Card
+      sx={{
+        m: 2,
+      }}>
+      <CardContent
+        sx={{
+          background: lightBlue,
+          display: "flex",
+          justifyContent: "space-between",
+          paddingBottom: "0 !important",
+        }}>
+        <Box overflow="hidden">
+          <Typography variant="h6">{props.courseName}</Typography>
+          <Typography
+            variant="caption"
+            noWrap>
```

Comment:
```suggestion
            >
```
Longer Descriptions are cut if user is on a smaller display

Reply:
changed